src/test/regress/output/directory_table.source

```diff
@@ -2063,6 +2063,21 @@ SELECT relative_path, tag FROM dir_table4 ORDER BY 1;
 
 SAVEPOINT s2;
 ROLLBACK;
+-- test create directory table with location
+CREATE DIRECTORY TABLE dir_table_with_location TABLESPACE directory_tblspc WITH LOCATION '/test_dirtable';
+CREATE DIRECTORY TABLE dir_table_with_location2 TABLESPACE directory_tblspc WITH LOCATION '/test_dirtable'; -- error
+ERROR:  directory table path "pg_tblspc//test_dirtable" already exists
```

Comment:
`pg_tblspc//test_dirtable` // is expected ?

Reply:
Yes，as we will append location string to pg_tblspc/， here the location is '/test_dirtable', so the 'pg_tblspc//test_dirtable' is expected which is also a valid path.